### PR TITLE
Suggest to add block for fixed statements

### PR DESCRIPTION
**Pull request by alrz**
and simplify code.

### CODE REVIEW

**Comment by alrz** [comment]
@sharwell @CyrusNajmabadi for review.

File: src/Features/CSharp/Portable/AddBraces/CSharpAddBracesCodeFixProvider.cs [hunk 1]
```diff
@@ -44,57 +44,17 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
                 // of other replace calls.  i.e. we may have statements nested in statements,
                 // we need to make sure that any inner edits are seen when we make the outer
                 // replacement.
-                editor.ReplaceNode(statement, (s, g) => GetReplacementNode(s));
+                editor.ReplaceNode(statement, (currentStatement, g) =>
+                {
+                    var embeddedStatement = currentStatement.GetEmbeddedStatement();
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/AddBraces/CSharpAddBracesCodeFixProvider.cs]: Nice.  i didn't know we had that extension.

**Comment by alrz** [comment]
@sharwell @CyrusNajmabadi This is left unlabeled. Can we merge this now?

**Comment by sharwell** [comment]
@alrz Thanks for the ping here. I added it to my radar for 15.6 (waiting for the build to pass).

**Pull request merged**